pkg/operator/sync.go

```diff
@@ -547,6 +547,33 @@ func newPodTemplateSpec(config *OperatorConfig, features map[string]bool) *corev
 	}
 }
 
+func getProxyArgs(config *OperatorConfig) []corev1.EnvVar {
+	var envVars []corev1.EnvVar
+
+	if config.Proxy != nil {
```

Comment:
This should be checking for `equal nil` not `not equal nil`
```suggestion
	if config.Proxy == nil {
```

Reply:
apologies, good catch there